models/prices/arbitrum/prices_arbitrum_tokens.sql

```diff
@@ -106,5 +106,26 @@ FROM
     ('oath-oath','arbitrum','OATH',0xa1150db5105987cec5fd092273d1e3cbb22b378b,18),
     ('winr-winr-protocol','arbitrum','WINR',0xd77b108d4f6cefaa0cae9506a934e825becca46e,18),
     ('ram-ramses-exchange','arbitrum','RAM',0xaaa6c1e32c55a7bfa8066a6fae9b42650f262418,18),
-    ('lusd-liquity-usd','arbitrum','LUSD',0x93b346b6BC2548dA6A1E7d98E9a421B42541425b,18)
+    ('lusd-liquity-usd','arbitrum','LUSD',0x93b346b6BC2548dA6A1E7d98E9a421B42541425b,18),
+    ('grail-camelot-token','arbitrum','GRAIL',0x3d9907f9a368ad0a51be60f7da3b97cf940982d8,18),
+    ('pnp-penpie','arbitrum','PNP',0x2ac2b254bc18cd4999f64773a966e4f4869c34ee,18),
+    ('premia-premia','arbitrum','PREMIA',0x51fc0f6660482ea73330e414efd7808811a57fa2,18),
+    ('kuji-kujira','arbitrum','KUJI',0x3a18dcc9745edcd1ef33ecb93b0b6eba5671e7ca,6),
+    ('rdnt-radiant-capital','arbitrum','RDNT',0x3082cc23568ea640225c2467653db90e9250aaa0,18),
+    ('euros-the-standard-euro','arbitrum','EUROS',0x643b34980e635719c15a2d4ce69571a258f940e9,18),
+    ('dmt1-dmt','arbitrum','DMT',0x8b0e6f19ee57089f7649a455d89d7bc6314d04e8,18),
+    ('pbx-paribus','arbitrum','PBX',0xbad58ed9b5f26a002ea250d7a60dc6729a4a2403,18),
+    ('pls-plutusdao','arbitrum','PLS',0x51318b7d00db7acc4026c88c3952b66278b6a67f,18),
+    ('pendle-pendle','arbitrum','PENDLE',0x0c880f6761f1af8d9aa9c466984b80dab9a8c9e8,18),
+    ('tig-tigris','arbitrum','TIG',0x3a33473d7990a605a88ac72a78ad4efc40a54adb,18),
+    ('aidoge-arbdoge-ai','arbitrum','AIDOGE',0x09e18590e8f76b6cf471b3cd75fe1a1a9d2b2c2b,6),
+    ('lode-lodestar','arbitrum','LODE',0xf19547f9ed24aa66b03c3a552d181ae334fbb8db,18),
+    ('hmx-hmx','arbitrum','HMX',0x83d6c8c06ac276465e4c92e7ac8c23740f435140,18),
+    ('aibb-bullbear-ai','arbitrum','AIBB',0xb9af4762c039d63e30039f1712dfab77026408c7,18),
+    ('dca-autodca','arbitrum','DCA',0x965f298e4ade51c0b0bb24e3369deb6c7d5b3951,18),
+    ('ush-unsheth','arbitrum','USH',0x51a80238b5738725128d3a3e06ab41c1d4c05c74,18),
+    ('hdn-hydranet','arbitrum','HDN',0x3404149e9ee6f17fb41db1ce593ee48fbdcd9506,18),
+    ('vela-vela-token','arbitrum','VELA',0x088cd8f5ef3652623c22d48b1605dcfe860cd704,18),
+    ('choke-artichoke','arbitrum','CHOKE',0x6fc2680d8ad8e8312191441b4eca9eff8d06b45a,18),
+    ('acid-0xacid','arbitrum','ACID',0x29c1ea5ed7af53094b1a79ef60d20641987c867e,18)
```

Comment:
`acid-0xacid` is not active:
https://api.coinpaprika.com/v1/coins/acid-0xacid
```suggestion
    ('choke-artichoke','arbitrum','CHOKE',0x6fc2680d8ad8e8312191441b4eca9eff8d06b45a,18)
```